metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -297,4 +327,49 @@ final class Doctor(
       )
     }
   }
+
+  private def extractTargetInfo(target: ScalaTarget) = {
```

Comment:
Can we introduce a case class that wraps all the columns here? It would be great to avoid passing around a `Tuple7`